modules/openy_features/openy_node/modules/openy_node_event/openy_node_event.info.yml

```diff
@@ -0,0 +1,29 @@
+name: 'OpenY Node Event'
+description: 'Event-related OpenY site configuration'
+type: module
+package: 'YMCA Maryland'
+core: 8.x
+dependencies:
```

Comment:
@podarok according to the documentation https://www.drupal.org/docs/8/creating-custom-modules/let-drupal-8-know-about-your-module-with-an-infoyml-file version is not a required field for the custom module.
For the contrib modules the version is generated automatically.

Reply:
@kolesnikoff totally true
But there is a thing - no version - no ?version=8.x.x string for the module libraries. Which makes caching tough and creates inconveniences for deployments
If you can't do it. I'll do it on my end, but this means - slower reviews

Reply:
@podarok we can do everything:) it's not a problem to add the versioning. I just want to be sure that we understand what we are doing.
I understand when the version is added during each release of the module. 
But in 100% of cases when we work with custom modules, we don't change the version manually.
That's why I was a bit surprised about this particular case.

Reply:
Sorry for confuse
This issue we've faced recently, when you have no changed version - browsers used to keep old JS file in cache instead of pulling new one after deploy. Which leads to highly complicated debugging sessions, especially when there is a mobile browser.
So it is better to bump the version of the module and library every time you have made a change to avoid possible caching issues

Reply:
it makes sense. ok, will check JS libs versions better. thank you for explanations.